### DISCUSSION TITLE
Cold and dark state

### DISCUSSION THREAD

**Discussion by Frenchy55** [flybywiresim/aircraft#5886] [Help]
I have the most recent FBW Development version in my community folder.  No other files.
I am able to load the FBW aircraft (in cold/dark state)  External power showing (green) but when pressed no power or sound
Nothing on the above board lights up. Battery same issues.  Not sure what I am doing wrong ?
Latest build using the  installer....and a clean community folder (so no possible conflict with other files/mods.
Epad launches (newer version).  I have connections to simbrief and last flightplan created.
In summary the aircraft remains in cold/dark state.  Not sure what I am doing wrong.
Any help is much appreciated.
